features/karaf/openhab-addons/src/main/resources/header.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
should be just "features/openhab-addons" now

Reply:
sure. that would also solve the merge conflict. just started with an old copy of master